tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/mapper/CurriculumMembershipMapper.java

```diff
@@ -371,4 +350,21 @@ private Rotation rotationDtoToRotation(RotationDTO rotationDto) {
     }
     return result;
   }
+
+  /**
+   * Convert a trainingNumberDTO to a trainingNumber object.
+   *
+   * @param trainingNumberDTO the TrainingNumberDTO object to convert
+   * @return a TrainingNumber object
+   */
+  private TrainingNumber trainingNumberDTOToTrainingNumber(TrainingNumberDTO trainingNumberDTO) {
+    TrainingNumber result = null;
+    if (trainingNumberDTO != null) {
+      result = new TrainingNumber();
+      result.setId(trainingNumberDTO.getId());
+      result.setIntrepidId(trainingNumberDTO.getIntrepidId());
+      result.setTrainingNumber(trainingNumberDTO.getTrainingNumber());
+    }
+    return result;
+  }
```

Comment:
This looks like the method removed above.

Reply:
Yep